### DISCUSSION TITLE
Can I enable debug log of communication between the front end and the backend?

### DISCUSSION THREAD

**Discussion by melonedo** [fonsp/Pluto.jl#1254] [Pluto development]
I can see a pretty-printed log of client request to the back end like #998 if some internal error is thrown, but can I enable it for every request? Currently I resort to Firefox's builtin websocket inspector.

**Comment by Ellipse0934** [comment]
Maybe you want to print info off  `Webserver/WebServer.jl/process_ws_message` [here](https://github.com/fonsp/Pluto.jl/blob/main/src/webserver/WebServer.jl#L308)
